frontend/src/scenes/audit-logs/advancedActivityLogsLogic.tsx

```diff
@@ -0,0 +1,172 @@
+                        const endDate = dateStringToDayJs(values.filters.end_date)
+                        if (endDate) {
+                            params.append('end_date', endDate.toISOString())
+                        }
+                    }
+
+                    values.filters.users.forEach((user) => params.append('users', user))
+                    values.filters.scopes.forEach((scope) => params.append('scopes', scope))
+                    values.filters.activities.forEach((activity) => params.append('activities', activity))
+
+                    params.append('page', values.currentPage.toString())
+                    params.append('page_size', ADVANCED_ACTIVITY_PAGE_SIZE.toString())
+
+                    const response = await api.get(`api/projects/@current/advanced_activity_logs/?${params}`)
+                    return response
+                },
+            },
+        ],
+
+        availableFilters: [
+            null as AvailableFilters | null,
+            {
+                loadAvailableFilters: async () => {
+                    const response = await api.get(`api/projects/@current/advanced_activity_logs/available_filters/`)
+                    return response
+                },
+            },
+        ],
+    })),
+
+    selectors(({ actions }) => ({
+        isFeatureFlagEnabled: [
+            (s) => [s.featureFlags],
+            (featureFlags): boolean => !!featureFlags[FEATURE_FLAGS.ADVANCED_ACTIVITY_LOGS],
+        ],
+
+        hasActiveFilters: [
+            (s) => [s.filters],
+            (filters: AdvancedActivityLogFilters): boolean => {
+                return (
+                    filters.start_date !== null ||
+                    filters.end_date !== null ||
+                    filters.users.length > 0 ||
+                    filters.scopes.length > 0 ||
+                    filters.activities.length > 0
+                )
+            },
+        ],
+
+        pagination: [
+            (s) => [s.currentPage, s.advancedActivityLogs],
+            (
+                currentPage: number,
+                advancedActivityLogs: CountedPaginatedResponse<ActivityLogItem>
+            ): PaginationManual => ({
+                controlled: true,
+                pageSize: ADVANCED_ACTIVITY_PAGE_SIZE,
+                currentPage,
+                entryCount: advancedActivityLogs.count || 0,
+                onBackward: () => actions.setPage(currentPage - 1),
+                onForward: () => actions.setPage(currentPage + 1),
+            }),
+        ],
+    })),
+
+    listeners(({ actions }) => ({
+        setPage: () => {
+            actions.loadAdvancedActivityLogs()
+        },
+        setFilters: () => {
+            actions.loadAdvancedActivityLogs()
+        },
+        clearAllFilters: () => {
+            actions.loadAdvancedActivityLogs()
+        },
+    })),
+
+    events(({ actions, values }) => ({
+        afterMount: () => {
+            if (!values.isFeatureFlagEnabled) {
```

Comment:
might be able to do this before mount so it doesn't flash? Or do it in the scene?